benchmarks/web-serving/memcached_server/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM ubuntu:14.04
-MAINTAINER Nooshin Mirzadeh <nooshin.mirzadeh@epfl.ch>
+MAINTAINER Arash Pourhabibi <arash.pourhabibi@epfl.ch>
```

Comment:
I believe Mark should be the maintainer of web-serving (all the tiers).

Reply:
I have this in a local commit which I can merge (all of the updated Dockerfiles have my email as well as the LABEL tag)

Reply:
@neo-apz done, please check